src/test/mir-opt/inline-closure.rs

```diff
@@ -36,7 +36,7 @@ fn foo<T: Copy>(_t: T, q: i32) -> i32 {
 //     _5 = (move _6, move _7);
 //     _8 = move (_5.0: i32);
 //     _9 = move (_5.1: i32);
-//     _0 = move _8;
+//     _0 = _8;
```

Comment:
I am very confused as to why this happened. Why was it a move before and not after?